app/app.go

```diff
@@ -183,6 +186,7 @@ type App struct {
 	CircuitKeeper       circuitkeeper.Keeper
 	HyperlaneKeeper     hyperlanekeeper.Keeper
 	WarpKeeper          warpkeeper.Keeper
+	ValaddrKeeper       valaddrkeeper.Keeper
```

Comment:
nit: but looks more readable this way
```suggestion
	ValAddrKeeper       valaddrkeeper.Keeper
```